toto/util.py

```diff
@@ -97,6 +97,15 @@ def import_rsa_key_from_file(filepath, password=None):
 
   return rsa_key
 
+def import_rsa_public_keys_from_files_as_dict(filepaths):
+  """Takes a list of filepaths to RSA public keys and returns them as a
+  dictionary conformant with ssl_crypto.formats.KEYDICT_SCHEMA."""
+  key_dict = {}
+  for filepath in filepaths:
+    key = import_rsa_key_from_file(filepath)
+    keyid = key['keyid']
+    key_dict[keyid] = key
+  return key_dict
```

Comment:
Yep. I think it is fine to just pipe through the exceptions from `import_rsa_key_from_file`. It will complain if the caller passes an `iterable` that does not contain file paths to keys.
We can use  a custom class based exception that is raised if the caller doesn't pass an `iterable`. But I'd prefer doing this here: https://github.com/in-toto/in-toto/issues/6#issuecomment-256983523